bigtable-hbase-parent/bigtable-hbase-mapreduce/pom.xml

```diff
@@ -137,6 +137,10 @@ limitations under the License.
                     <groupId>org.slf4j</groupId>
                     <artifactId>slf4j-api</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
Hmm, that's a bit weird...maven's shade plugin should already be excluding the shaded deps from the resulting pom for bigtable-hbase-1.2. I was guessing that eclipse wasn't parsing the transformed pom.xml and thats why the exclusion was necessary.

Without this change, running `mvn dependency:tree` in bigtable-hbase-mapreduce I get the following:

> [INFO] +- com.google.cloud.bigtable:bigtable-hbase-1.2:jar:0.9.5-SNAPSHOT:compile
> [INFO] |  +- org.apache.hbase:hbase-client:jar:1.2.4:compile
> [INFO] |  |  +- org.apache.hbase:hbase-annotations:jar:1.2.4:compile
> [INFO] |  |  +- org.apache.hbase:hbase-protocol:jar:1.2.4:compile
> [INFO] |  |  \- com.yammer.metrics:metrics-core:jar:2.2.0:compile
> [INFO] |  \- org.apache.hbase:hbase-common:jar:1.2.4:compile
> [INFO] |     \- com.github.stephenc.findbugs:findbugs-annotations:jar:1.3.9-1:compile